job_spec/INSAR_ISCE_BURST.yml

```diff
@@ -0,0 +1,53 @@
+INSAR_ISCE_BURST:
+  required_parameters:
+    - granules
+  parameters:
+    granules:
+      default:  '""'
+      api_schema:
+        type: array
+        minItems: 2
+        maxItems: 2
+        example:
+          - S1_249434_IW1_20230523T170733_VV_8850-BURST
+          - S1_249434_IW1_20230511T170732_VV_07DE-BURST
+        items:
+          description: The reference and secondary bursts.
+          type: string
+          pattern: "^S1_.*-BURST"
```

Comment:
@forrestfwilliams I think we should only support `IW` burst granules, and exclude `EW`?

Reply:
This is included in the future goals listed in the JIRA ticket.